src/frequenz/client/microgrid/metrics/_metric.py

```diff
@@ -0,0 +1,238 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Supported metrics for microgrid components."""
+
+
+import enum
+
+
+@enum.unique
+class Metric(enum.Enum):
+    """List of supported metrics.
+
+    Note: AC energy metrics information
+        - This energy metric is reported directly from the component, and not a
+          result of aggregations in our systems. If a component does not have this
+          metric, this field cannot be populated.
+
+        - Components that provide energy metrics reset this metric from time to
+          time. This behaviour is specific to each component model. E.g., some
+          components reset it on UTC 00:00:00.
+
+        - This energy metric does not specify the timestamp since when the energy
+          was being accumulated, and therefore can be inconsistent.
+    """
+
+    UNSPECIFIED = 0
+    """The metric is unspecified (this should not be used)."""
+
+    DC_VOLTAGE = 1
```

Comment:
Why numbers?  This could be `pb_something.Metric.DV_VOLTAGE` right?

Reply:
Yes, I actually went back and forth about this. The reason was to avoid exposing any PB stuff in the client (and since we are supposed to always be backwards-compatible with the APIs, these numbers should never ever change, so it should be a one time thing only). A minor reason I think it was also that using the full PB symbols sometimes didn't even fit in one line :laughing: 

Import-wise, with the current file structure, you shouldn't need to import (even indirectly) any PB stuff if you are only using wrappers (the PB stuff should be only pulled if using the client itself). Since at the end you'll probably end up importing everything anyway, in practice it probably doesn't matter.

Also I was thinking also about the option of exposing the raw PB message, in case you are interacting with a newer version of the server, so you can potentially use new fields that are not supported by the client, so I'm open to go back to use the PB symbolic values. If that's the case it wasn't a completely wasteful exercise as at least I found some [unexpected gaps](https://github.com/frequenz-floss/frequenz-api-common/issues/254) and [inconsistentcies](https://github.com/frequenz-floss/frequenz-api-common/issues/253) in the tags.